.devcontainer/post-create.sh

```diff
@@ -0,0 +1,5 @@
+curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sudo sh -s -- -b /usr/local/bin v0.48.2
```

Comment:
can we install from apt or github release instead of `curl | sudo sh`

we can also add this to Makefile for a devcontainer specific target and have version as a variable so we don't have to update in multiple places in the future